src/proof/er/er_proof.cpp

```diff
@@ -126,12 +127,12 @@ ErProof ErProof::fromBinaryDratProof(const ClauseUseRecord& usedClauses,
   // Parse the resulting TRACECHECK proof into an ER proof.
   std::ifstream tracecheckStream(tracecheckFilename);
   ErProof proof(usedClauses, TraceCheckProof::fromText(tracecheckStream));
-  unlink(tracecheckFilename);
-
-  formStream.close();
-  dratStream.close();
   tracecheckStream.close();
 
+  unlink(formulaFilename);
```

Comment:
Nit: `remove()` is standard C while `unlink()` is POSIX, so it is a bit better to use `remove()` here.

Reply:
Sounds good! Will do.